package.json

```diff
@@ -20,8 +20,8 @@
     "migrate": "IASQL_ENV=ci ts-node ./node_modules/.bin/typeorm migration:run",
     "gen-module": "IASQL_ENV=ci ./.module_migration.sh",
     "docker-build": "docker build -t iasql:latest .",
-    "docker-run": "docker run -e IASQL_ENV=local -p 5432:5432 --name iasql iasql:latest",
-    "docker-run-vol": "docker run -e IASQL_ENV=local -p 5432:5432 -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
```

Comment:
is local the default now?

Reply:
Yes, it is. I did that in anticipation of us eventually publishing to Dockerhub and people just running `docker run -p 5432:5432 iasql:latest` and having it Just Work :tm: